src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientHeaderSampleComposer.java

```diff
@@ -81,7 +87,8 @@ public static Sample composeShowcaseMethodSample(
       TypeNode clientType,
       List<MethodArgument> arguments,
       Map<String, ResourceName> resourceNames,
-      Map<String, Message> messageTypes) {
+      Map<String, Message> messageTypes,
+      Service service) {
```

Comment:
Do we still need to pass in Service if we are parsing Strings to get the service name?

Reply:
Thanks for catching - when I switched approaches to instead parse the service name, I didn't do a thorough job cleaning up. Updated!